README.rst

```diff
@@ -72,3 +72,4 @@ Before sending a pull request, please ensure the following:
 * The changelog has been updated, to indicate the change made along
   with the issue no. on GitHub. The changelog is to be in `releases
   format <http://releases.readthedocs.org/en/latest/index.html>`_.
+
```

Comment:
There still seems to be some change made here @Anupama-Github. Request you to revert this file so that other changes made to `README.rst` will be brought in on merge